frontend/src/components/Wallet/LoginModal.tsx

```diff
@@ -54,13 +61,13 @@ const LoginModal: React.FC<IProps> = ({ onLogin, ...rest }) => {
   const loginTypes = [
     {
       title: "Fuelet",
-      isActive: window.fuelet != null,
+      isActive: window.fuelet !== undefined,
```

Comment:
We use checks for null, not undefined, cos this object can be also null
And this check is the way longer too, we prefer  shorter solutions

Reply:
also != null checks for all empty values (even empty object works)